eng/tools/generator/cmd/issue/issueCmd.go

```diff
@@ -116,17 +116,17 @@ func (c *commandContext) execute() error {
 	if err != nil {
 		return err
 	}
+	// we first output the config to stdout, then validate it so that the user could always get a usable config
+	// validate the config
```

Comment:
since you change the order, then this comment is not valid.

Reply:
I change the comment.and move it above the output line